.github/workflows/tests.yml

```
@@ -189,12 +189,7 @@ jobs:
           macOS)
             brew update
             export HOMEBREW_NO_INSTALL_UPGRADE=1 HOMEBREW_NO_INSTALLED_DEPENDENTS_CHECK=1
-            brew install ccache ghostscript ninja
-            # The macOS 12 images have an older Python, and this causes homebrew to generate conflicts.
-            # We'll just skip GTK for now, to not pull in Python updates.
-            if [[ "${{ matrix.os }}" = macos-14 ]]; then
-              brew install gobject-introspection gtk4
-            fi
+            brew install ccache ghostscript gobject-introspection gtk4 ninja
```

```suggestion
            brew install ccache ghostscript ninja
```

will this skip the Python update?

---

Ah right, maybe I'll put the condition back in.